### ISSUE TITLE
Ensemble forecast output

### ISSUE THREAD

**Issue by haseebmahmud** [rafzamb/sknifedatar#22]
Is there any possibility that I can combine output of multiple forecast into one?

**Comment by mabuimo** [comment]
Yes, you can create ensemble models. An option is to use [modeltime.ensemble](https://business-science.github.io/modeltime.ensemble/).
